src/tests/Blef.GameLogic.Tests/PokerHands/HighCardTests.cs

```diff
@@ -41,5 +41,16 @@ public void Given_table_with_many_high_cards_Then_result_is_True()
 
             Assert.True(isOnTable);
         }
+
+        [Fact]
+        public void When_comparing_better_hand_to_worse_hand_then_result_is_bigger_than_zero()
+        {
+            HighCard betterHand = new HighCard(Card.Jack);
+            HighCard worseHand = new HighCard(Card.Nine);
+
+            int result = betterHand.CompareTo(worseHand);
```

Comment:
Instead of returning magic numbers (-1, 0. 1) please consider introduction of enum with domain meaning (lower, higher, equal). You should add at least three tests for this functionality. How about names like:
should_be_able_to_compare_two_high_card_when_first_is_higher()
should_be_able_to_compare_two_high_card_when_first_is_lower()
should_be_able_to_compare_two_high_card_when_both_are_equal()